### PR TITLE
add menu item

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@smn @bruskiza ready for review

File: tuneme/templates/_base.html [hunk 1]
```diff
@@ -26,41 +26,50 @@
...
-          loadJS( "{% static 'js/main.js' %}" );
+    	  // Show this item if it was previously hidden
+          if ( itemClass == 'menuItem hide' ) {
+            this.parentNode.className = 'menuItem';
+          }
+        }
+        function getFirstChildWithTagName( element, tagName ) {
+          for ( var i = 0; i < element.childNodes.length; i++ ) {
+            if ( element.childNodes[i].nodeName == tagName ) return element.childNodes[i];
+          }
```
Comment (smn) [inline tuneme/templates/_base.html:64]: I think we've thrown away more javascript than we'd like here. Not sure where things were copied & pasted from?

**Comment by smn** [comment]
:+1: but I'm a bit concerned about the javascript that was removed. I suspect we're removing more than we'd like. Please create an issue for this specifically so we can look back and restore if needed, remember to link to this pull-request please.

Nice work @saeedmb8 !

File: tuneme/templates/_base.html (hunk 1 above)
Comment (smarzban) [inline tuneme/templates/_base.html:64]: @smn I’ve copied the javascript from the github page I’ve checked with milton’s work in the bwise he also removed them all so I assumed we don’t need them.

**Comment by smarzban** [comment]
@smn thanks, I've created issue-18 for it.

**Comment by smn** [comment]
thanks!

**Pull request merged**